backbone/src/main/java/org/researchstack/backbone/ui/step/layout/SurveyStepLayout.java

```diff
@@ -153,12 +154,12 @@ public void onClick(View view)
         });
 
         if (questionStep != null) {
-            if (!TextUtils.isEmpty(questionStep.getTitle())) {
+            if (questionStep.getStepBodyClass() == FormBody.class && !TextUtils.isEmpty(questionStep.getTitle())) {
```

Comment:
why do we need this check now?

Reply:
We don't use a custom Axon step for Form steps. As such, we need to display the native RS Title and Description for Form steps...and only for Form steps.